exercises/dnd-character/.meta/template.j2

```diff
@@ -0,0 +1,33 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% set class = exercise | camel_case -%}
+{{ macros.header(["Character", "modifier"]) }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases[0]["cases"] -%}
+    def test_{{ case["description"] | replace("-","n") | to_snake }}(self):
+        {% set value = case["input"]["score"] -%}
+        {% set expected = case["expected"] -%}
+        self.assertEqual({{ case["property"] }}({{ value }}), {{ expected }})
+
+    {% endfor %}
+
+    def test_random_ability_is_within_range(self):
+        score = Character().{{ cases[1]["property"] }}()
```

Comment:
What if another case is inserted before this one?

Reply:
Likely we'll get a bug and then we can fix it. I mean it's not easy to anticipate all the changes of the test file to cover them all in the template. There's many way we can ask "What if" already: What if the depth is 3 or higher, what if a new test case that raises an error is added to the list of normal test cases,...

Btw, what change do you suggest for this one?

Reply:
I'm not sure at the moment... give me some time to think on it.